deploy/deploy.go

```diff
@@ -43,6 +43,7 @@ var ProtoMeta = types.ProtocolMeta{
 	Router:                   common.HexToAddress("0xe6e6e71E747EeD9fe45F0ff63E1B3E4a7c1199bF"),
 	WFIL:                     common.HexToAddress("0x60E1773636CF5E4A227d9AC24F20fEca034ee25A"),
 	GLF:                      common.HexToAddress("0x2a0aaf86b2FA64E88D3739091e7773106F3ebCF5"),
+	// Plus:                     common.HexToAddress("FIXME"),
```

Comment:
For now, let's just add this Plus as `common.HexToAddress("0x0")` (just use an empty address). 

I think this needs to be applied twice - once for the test contracts, second for the mainnet ones